hazelcast/src/main/java/com/hazelcast/map/impl/operation/AddIndexBackupOperation.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.map.impl.operation;
+
+import com.hazelcast.core.Member;
+import com.hazelcast.map.impl.MapDataSerializerHook;
+import com.hazelcast.map.impl.MapService;
+import com.hazelcast.nio.Address;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.query.impl.Indexes;
+import com.hazelcast.spi.BackupOperation;
+import com.hazelcast.spi.impl.operationservice.TargetAware;
+import com.hazelcast.version.MemberVersion;
+
+import java.io.IOException;
+
+public class AddIndexBackupOperation extends MapOperation implements BackupOperation, TargetAware {
+
+    private static final MemberVersion V3_12_1 = MemberVersion.of(3, 12, 1);
+
+    private String attributeName;
+    private boolean ordered;
+
+    // true by default just to make DataSerializableConventionsTest pass
+    private boolean targetSupported = true;
```

Comment:
This field is `transient`.
Also, I'd rather have `AddIndexBackupOperation` added to the [DataSerializableConventionsTest whitelist](https://github.com/hazelcast/hazelcast/blob/23874cba403df9667e56ef2f45d1c99f9c8ed7aa/hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/DataSerializableConventionsTest.java#L72-L77) than having `true` as default here.

Reply:
fixed